File: samples/client/petstore/java/src/main/java/org/openapijsonschematools/client/apis/paths/Anotherfakedummy.java
package org.openapijsonschematools.client.apis.paths;

import org.openapijsonschematools.client.apiclient.ApiClient;
import org.openapijsonschematools.client.configurations.ApiConfiguration;
import org.openapijsonschematools.client.configurations.SchemaConfiguration;
import org.openapijsonschematools.client.paths.anotherfakedummy.Patch;

public class Anotherfakedummy extends ApiClient implements
    Patch.PatchOperation
{
    public Anotherfakedummy(ApiConfiguration apiConfiguration, SchemaConfiguration schemaConfiguration) {
        super(apiConfiguration, schemaConfiguration);
    }
}

File: samples/client/petstore/kotlin/src/main/kotlin/org/openapijsonschematools/client/apis/paths/Anotherfakedummy.kt
package org.openapijsonschematools.client.apis.paths;

import org.openapijsonschematools.client.apiclient.ApiClient;
import org.openapijsonschematools.client.configurations.ApiConfiguration;
import org.openapijsonschematools.client.configurations.SchemaConfiguration;
import org.openapijsonschematools.client.paths.anotherfakedummy.AnotherfakedummyPatch;

public class Anotherfakedummy extends ApiClient implements
    AnotherfakedummyPatch.PatchOperation
{
    public Anotherfakedummy(ApiConfiguration apiConfiguration, SchemaConfiguration schemaConfiguration) {
        super(apiConfiguration, schemaConfiguration);
    }
}